huntsman/utils/config.py

```diff
@@ -151,17 +148,30 @@ def query_config_server(key=None, name='config_server', logger=None):
     if logger is None:
         logger = DummyLogger()
         
-    try:
-        config_server = Pyro4.Proxy(f'PYRONAME:{name}')
-        return config_server.get_config(key=key)
-    
-    except Exception as e:
-        logger.error(f'Unable to load remote config: {e}')
-        raise(e)
+    while True:
+        
+        try:
+            config_server = Pyro4.Proxy(f'PYRONAME:{name}')
+            return config_server.get_config(key=key)
+        
+        except Pyro4.errors.NamingError as e:
+            
+            if wait is not None:
+                logger.info('Failed to locate config server. \
+                            Waiting %is before retrying.'%wait)
```

Comment:
It's fine to use the logger style formatting but in general it's better to be consistent and just use f-strings or `.format`. It's true that there are some slight advantages to letting the logger do it but it's not as robust.

Reply:
Interesting what you say about PANDIR, although I don't quite understand. Where does e.g. Huntsman-POCS get installed in the container if PANDIR is (presumably) a mounted volume? I will take a look at the PANOPTES stuff. 

As @lspitler mentioned, there was some weird error while mounting the SSHFS and this was a temporary work around. Will take a look, but perhaps this can be part of the next PR in which the docker stuff is cleaned up a little (perhaps including docker-compose, which I need to read up on).